src/screens/farming/constants.ts

```diff
@@ -1,3 +1,2 @@
 export const DEFAULT_AMOUNT = 0;
-export const DEFAULT_DECIMALS = 2;
 export const PENNY = 0.01;
```

Comment:
we have the same constants  in `src/components/earn-opportunities-main-info/constants.ts`

maybe better to reuse one of it?